src/components/LoginCard.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <!-- WORK IN PROGRESS-->
+  <form novalidate @submit.prevent="" class="login-card">
+    <BaseInput @input="setInput" class="login-card__email" />
```

Comment:
you can already use `v-model` here, because you did a great job with a `BaseInput` component